server/src/graql/parser/Parser.java

```diff
@@ -409,7 +411,7 @@ public Pattern visitPattern(GraqlParser.PatternContext ctx) {
     }
 
     @Override
-    public Disjunction<?> visitPattern_disjunction(GraqlParser.Pattern_disjunctionContext ctx) {
+    public Pattern visitPattern_disjunction(GraqlParser.Pattern_disjunctionContext ctx) {
```

Comment:
It's good practice to return the strict type whenever known. So this method should be returning `Disjunction<?>`.